data/ka'het missions.txt

```diff
@@ -89,4 +89,130 @@ mission "First Contact: Ka'het: Remnant 1B"
+	to fail
+		has "First Contact: Ka'het: Quarg 2: offered"
+	on offer
+		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."
+		conversation
+			`Due to your past experience, you expected to crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
+			`	More than the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to go ask the Quarg what happened here...`
+				accept
+
+mission "First Contact: Ka'het: Quarg 1D"
+	name "Abandoned Ringworld"
+	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	landing
+	source "Querii Ka"
+	destination "Lagrange"
+	stopover "Kuwaru Efreti"
+	stopover "Alta Hai"
+	to offer
+		not "First Contact: Ka'het: Quarg 1A: offered"
+		not "First Contact: Ka'het: Quarg 1B: offered"
+		not "First Contact: Ka'het: Quarg 1C: offered"
+		has "First Contact: Quarg: offered"
+		has "First Contact: Hai: offered"
+		has "First Contact: Kuwaru Efreti: offered"
+	to fail
+		has "First Contact: Ka'het: Quarg 2: offered"
+	on offer
+		log "Discovered an uninhabited, complete Quarg ringworld in the middle of Ka'het space."
+		conversation
+			`Due to your past experience, you expected to crowds of the tall and inexpressive Quarg walking near the landing pads of this ringworld; instead, the whole station seems uninhabited.`
+			`	More than the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to go ask the Quarg what happened here...`
+				accept
+
+mission "First Contact: Ka'het: Quarg 2"
+	landing
+	invisible
+	source
+		planet "Alta Hai" "Lagrange" "Kuwaru Efreti"
+	to offer
+		or
+			has "First Contact: Ka'het: Quarg 1A: active"
+			has "First Contact: Ka'het: Quarg 1B: active"
+			has "First Contact: Ka'het: Quarg 1C: active"
+			has "First Contact: Ka'het: Quarg 1D: active"
+	on offer
+		conversation
+			`Do you want to ask the Quarg about their abandoned ringworld in the Graveyard?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	Walking down the principal, alien-looking hall, you decide to stop one of the few Quarg you see there. "Hello, human. What help do you need?" he says, and you reply asking him what happened in the Graveyard and to the ringworld they built there.`
+			`	"Ah, you encountered the remainder of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they refrained from accepting our aiding, and attempted to relocate at a greater distance from us."`
+			choice
+				`	"Who were the hostile contacts with?"`
```

Comment:
`	"Who were these hostile contacts?"`

Reply:
I'm not sure this is better than what we currently have

Reply:
Doesn't seem like an entirely necessary change, but I do somewhat prefer this suggestion.

Reply:
Alright, changed